File: Serene/Serene.Script/Meeting/MeetingDecisionRelevant/MeetingDecisionRelevantDialog.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System.Collections.Generic;

    [IdProperty("MeetingDecisionRelevantId"), IsActiveProperty("IsActive")]
    [FormKey("Meeting.MeetingDecisionRelevant"), LocalTextPrefix("Meeting.MeetingDecisionRelevant"), Service("Meeting/MeetingDecisionRelevant")]
    public class MeetingDecisionRelevantDialog : EntityDialog<MeetingDecisionRelevantRow>
    {
    }
}

File: Serene/Serene.Web/Modules/Meeting/MeetingDecisionRelevant/MeetingDecisionRelevantDialog.ts

namespace Serene.Meeting {

    @Serenity.Decorators.registerClass()
    @Serenity.Decorators.responsive()
    export class MeetingDecisionRelevantDialog extends Serenity.EntityDialog<MeetingDecisionRelevantRow, any> {
        protected getFormKey() { return MeetingDecisionRelevantForm.formKey; }
        protected getIdProperty() { return MeetingDecisionRelevantRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingDecisionRelevantRow.localTextPrefix; }
        protected getService() { return MeetingDecisionRelevantService.baseUrl; }

        protected form = new MeetingDecisionRelevantForm(this.idPrefix);

    }
}